ci/Dockerfile.manylinux.cpu

```diff
@@ -0,0 +1,57 @@
+# Taichi Dockerfile (CPU only) for Manylinux2014 compliant
+FROM quay.io/pypa/manylinux2014_x86_64 
+
+LABEL maintainer="https://github.com/taichi-dev"
+
+RUN yum check-update && \
+    yum install -y git \
+                   cmake \
+                   wget \
+                   libXrandr
+
+# Build LLVM/Clang 10 from source
+WORKDIR /
+RUN wget https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/llvm-10.0.0.src.tar.xz
+RUN tar -xf llvm-10.0.0.src.tar.xz && \
+    rm llvm-10.0.0.src.tar.xz
+RUN wget https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/clang-10.0.0.src.tar.xz
+RUN tar -xf clang-10.0.0.src.tar.xz && \
+    rm clang-10.0.0.src.tar.xz
+RUN cp -r clang-10.0.0.src llvm-10.0.0.src/tools/clang
+
+WORKDIR /llvm-10.0.0.src/build
+RUN cmake .. -DLLVM_ENABLE_RTTI:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=OFF -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86" -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_ENABLE_TERMINFO=OFF
+RUN make -j 8 && \
+    make install
+ENV CC="/usr/local/bin/clang"
+ENV CXX="/usr/local/bin/clang++"
+
+# Link gcc 10 to build Taichi
+WORKDIR /usr/lib/gcc/x86_64-redhat-linux/
+RUN ln -s /opt/rh/devtoolset-10/root/usr/lib/gcc/x86_64-redhat-linux/10 10
+# Check gcc-10 is used
+RUN clang++ -v
+
+# Create non-root user for running the container
+RUN useradd -ms /bin/bash dev
+WORKDIR /home/dev
+USER dev
+
+# Install miniconda
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    bash Miniconda3-latest-Linux-x86_64.sh -p /home/dev/miniconda -b
+ENV PATH="/home/dev/miniconda/bin:$PATH"
+
+# Set up multi-python environment
+RUN conda init bash
+RUN conda create -n py36 python=3.6 -y
```

Comment:
FYI, Python 3.6-3.10 is already installed in /`opt/python` so conda may not be necessary.

Reply:
@messense, thx, good to know that!